app/docs/src/how-tos/aws-eks.md

```diff
@@ -0,0 +1,520 @@
+Role frame.
+
+Configure it as follows:
+
+| Component Name                     | `PolicyArn`                                                |
+| ---------------------------------- | ---------------------------------------------------------- |
+| AmazonEKS_CNI_Policy               | arn:aws:iam::aws:policy/AmazonEKS_CNI_Policy               |
+| AmazonEC2ContainerRegistryReadOnly | arn:aws:iam::aws:policy/AmazonEC2ContainerRegistryReadOnly |
+| AmazonEKSWorkerNodePolicy          | arn:aws:iam::aws:policy/AmazonEKSWorkerNodePolicy          |
+
+Connect the `RoleName` output socket of the `nodegroup-profile-role` to the
+`RoleName` input socket of each of the AWS IAM Role Policy components in that
+frame.
+
+### Create a Security Group component for the NodeGroup
+
+![Create Security Group](./aws-eks/create-ec2-security-group-for-nodegroup.png)
+
+Add a `Security Group` to your `demo-eks-cluster` eks cluster frame.
+
+Set the component name to `demo-eks-nodegroup-sg`.
+
+Set the `GroupName` to `demo-eks-nodegroup-sg`.
+
+Set the `Description` to be `Security Group for the EKS NodeGroup`
+
+Connect the `Security Group ID` output socket of `demo-eks-nodegroup-sg`
+component to the `Security Group ID` input socket of the `demo-nodegroup` frame.
+
+### Create an Ingress Rule component
+
+![Create Ingress Rule](./aws-eks/create-security-group-ingress-for-nodegroup.png)
+
+Add a `Security Group Rule (Ingress)` to your `demo-eks-cluster` eks cluster
+frame.
+
+Set the component name to be `nodegroup-all-protocol-all-ports`.
+
+Set the `Description` to be
+`Ingress to allow all ports and protocols on the NodeGroup`.
+
+Set the `TrafficPort` to be `-1/-1`.
+
+Connect the `Security Group ID` output socket of `demo-eks-nodegroup-sg`
+component to the `Security Group ID` input socket of this
+`nodegroup-all-protocol-all-ports` component.
+
+Connect the `Security Group ID` output socket of
+eks-cluster-sg`component to the`Source Traffic Security Group
+ID`input socket of this`nodegroup-all-protocol-all-ports` component.
+
+### Apply your Change Set
+
+![Apply Change Set](./aws-eks/apply.png)
+
+Press `Escape` or click anywhere on the canvas background to select the
+Workspace.
+
+Click the `Apply Change Set` button to:
+
+- Create all of the desired IAM Roles and Policies
+- Create all of the Security Groups
+- Create an EKS Fargate Profile
+- Create an EKS NodeGroup
+
+NOTE: When creating a nodegroup and/or a fargate profile, AWS requires the
+cluster to be `ACTIVE`. The create actions will fail with `warning` messages.
+You can retry these actions after a period of time.
+
+## Accessing the cluster
+
+By default, the user used to create the cluster will be the user to have access
+to it. It is not recommended to use access keys directly with System Initiative.
+The workspace uses assumeRolePolicy, so specific users will need granded access
+to the cluster.
+
+### Generating a KubeConfig
+
+Using the AWS CLI, run the following command:
```

Comment:
There's nothing here to say get a set of credentials into AWS, basically it might be worth saying to the reader to setup some creds which you want to authenticate to the cluster & you can check who the user is with `aws sts get-caller-identity`

Reply:
I reckon a lot of users `will not` have a valid set at this stage

Reply:
I have linked to the AWS CLI docs and the configuring it